### PR TITLE
chore: improve get started

### PR DESCRIPTION
**Pull request by floridemai**
Getting started page was pretty empty and we choose to load the readme file and use that.

Options:
* "start" with the readme and create another example section as per the ticked
* simply add a how to use in the readme and just place the readme in the get started page
(The tricky part is that it's a get started while the readme starts with a header and the name)

Solution:
 I see what you mean, but considering the simplicity of the solution that embeds the readme, I'm happy with including the raw readme in there. Let's expand the readme with a simple usage section and we're done

### CODE REVIEW

File: stories/GetStarted.mdx [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 import { ColorWrapper } from './colorUtils.tsx'
+import ReadMe from '../README.md?raw'
 
 <Meta title="Get Started" />
 
-<h1>Get Started</h1>
-
-<p>Welcome to Quantum. This guide will help you to get up and running in no time.</p>
+<Markdown>
+  {ReadMe.replace(/!\[CI\]\(.*\)\s*\[!\[Figma\]\(.*\)\]\(.*\)/, '')}
```
Comment (simoneb) [inline stories/GetStarted.mdx:10]: will the badges not work within storybook?

Comment (floridemai) [reply stories/GetStarted.mdx:10]: image
This is what caught my attention. Should I look at a solution for having both in there?

Comment (simoneb) [reply stories/GetStarted.mdx:10]: ideally yes, weird that the firma badge doesn't reneder though, it looks like valid markdown that storybook should be able to render

Comment (floridemai) [reply stories/GetStarted.mdx]: @simoneb I tried latest version of Markdown from storybook/blocks but it didn't work either so I used react-markdown (this was my initial lib anyway) which seems to be rendering it ok (does a new line not space, but that works I. guess)

**Pull request merged**